### PR TITLE
Remove revertRemovalOfSiblingPrerendering killswitch

### PR DESCRIPTION
**Pull request by sammy-SC**
removal of sibling prerendering has been rolled out at Meta. We can delete the flag now.

### CODE REVIEW

File: packages/react-reconciler/src/ReactFiberWorkLoop.js [hunk 1]
```diff
@@ -2535,28 +2534,14 @@ function completeUnitOfWork(unitOfWork: Fiber): void {
...
-        // The reason "unwind" and "complete" is interleaved is because when
-        // something suspends, we continue rendering the siblings even though
-        // they will be replaced by a fallback.
-        // TODO: Disable sibling prerendering, then remove this branch.
-        unwindUnitOfWork(completedWork);
-        return;
-      }
-    } else {
-      if (__DEV__) {
-        if ((completedWork.flags & Incomplete) !== NoFlags) {
```
Comment (acdlite) [inline packages/react-reconciler/src/ReactFiberWorkLoop.js:2552]: Can you please leave these else branches? At least for now while the code paths are still relatively new.

File: packages/react-reconciler/src/ReactFiberWorkLoop.js [hunk 2]
```diff
@@ -2660,25 +2645,6 @@ function unwindUnitOfWork(unitOfWork: Fiber): void {
...
-      // TODO: Don't prerender siblings. With `use`, we suspend the work loop
-      // until the data has resolved, anyway.
-      const siblingFiber = incompleteWork.sibling;
-      if (siblingFiber !== null) {
-        // This branch will return us to the normal work loop.
-        workInProgress = siblingFiber;
-        return;
-      }
-    } else {
-      // NOTE: If we re-enable sibling prerendering in some cases, this branch
```
Comment (acdlite) [inline packages/react-reconciler/src/ReactFiberWorkLoop.js:2677]: Same here, I'd like to keep this comment around for context

File: packages/react-reconciler/src/ReactFiberWorkLoop.js (hunk 1 above)
Comment (acdlite) [reply packages/react-reconciler/src/ReactFiberWorkLoop.js:2552]: Oh wait never mind you did leave this else branch. Ignore me 😆

File: packages/react-reconciler/src/ReactFiberWorkLoop.js (hunk 2 above)
Comment (sammy-SC) [reply packages/react-reconciler/src/ReactFiberWorkLoop.js:2677]: makes sense. I added the comment back.

**Review by acdlite — APPROVED** [review]

**Pull request merged**